process/block/shardinfo_test.go

```diff
@@ -1017,16 +1012,15 @@ func TestShardInfoCreateData_updateShardDataWithCrossShardInfo(t *testing.T) {
 		)
 		require.Nil(t, err)
 		err = sic.updateShardDataWithCrossShardInfo(shardData, &header)
-		assert.NotNil(t, shardData)
+		require.NotNil(t, shardData)
 		require.Nil(t, err)
-		assert.Equal(t, uint32(2), shardData.NumPendingMiniBlocks)
-		assert.Equal(t, uint64(expectedNonce), shardData.LastIncludedMetaNonce)
+		require.Equal(t, uint32(2), shardData.NumPendingMiniBlocks)
+		require.Equal(t, uint64(expectedNonce), shardData.LastIncludedMetaNonce)
```

Comment:
This cast is still redundant
```suggestion
		require.Equal(t, expectedNonce, shardData.LastIncludedMetaNonce)
```